packages/cf-component-checkbox/src/Checkbox.js

```diff
@@ -21,7 +21,7 @@ class Checkbox extends React.Component {
           name={this.props.name}
           value={this.props.value}
           checked={this.props.checked}
-          onChange={this.handleChange}/>
+          onChange={this.handleChange.bind(this)}/>
```

Comment:
Might as well get rid of `handleChange()` entirely and pass the `onChange` props directly to the input element without any binding. We'll up the major version number when we release.